OpenRA.Mods.Common/Traits/SupportPowers/SpawnActorPower.cs

```diff
@@ -50,30 +62,113 @@ public override void Activate(Actor self, Order order, SupportPowerManager manag
 			base.Activate(self, order, manager);
```

Comment:
We should probably call this after the checks, shouldn't we?

Reply:
Maybe.